pkg/reconciler/contour/resources/httpproxy_test.go

```diff
@@ -124,6 +124,7 @@ func TestMakeProxies(t *testing.T) {
 					PermitInsecure:   true,
 					TimeoutPolicy: &v1.TimeoutPolicy{
 						Response: "infinity",
+						Idle:     "infinity",
```

Comment:
Is this a behavior change, or are we just inlining the default behavior?

If a change, can you call our in a ```release-notes  section of the PR?

Reply:
Added